docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>The first phrase is strange. Consider rewording to:</p><p><br></p><p>"You now have the ability to train with different sets of hyperparameters. You'll now try all combinations ..."</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Indeed, saying "runs" three times in a sentence is not ideal.